pkgs/development/idris-modules/tfrandom.nix

```diff
@@ -0,0 +1,21 @@
+{ stdenv
+, build-idris-package
+, fetchFromGitHub
+, lib
+}:
+build-idris-package  {
+  name = "tf-random";
+  version = "2020-01-15";
+
+  src = fetchFromGitHub {
+    owner = "david-christiansen";
+    repo = "idris-tf-random";
+    rev = "202aac3b96757e8247f6e26945329d90dd668aed";
+    sha256 = "1z8pyrsm1kdsspcs3h96k38h44ss0mv39lcz7xvwg8ickys3kqxl";
+  };
+
+  meta = {
+    description = "Port of Haskell tf-random";
+    homepage = "https://github.com/david-christiansen/idris-tf-random";
```

Comment:
License missing, also if you want you can add yourself as a maintainer (see how other packages declare meta)